bucket/oss-browser.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.9.5",
+    "description": "ossbrowser is a graphical management tool developed by Alibaba Cloud. It provides features similar to those of Windows Explorer. Using ossbrowser, you can view, upload, download, and manage objects with ease.",
+    "homepage": "https://github.com/aliyun/oss-browser",
+    "license": "Apache License 2.0",
+    "url": "https://oss-attachment.cn-hangzhou.oss.aliyun-inc.com/oss-browser/1.9.5/oss-browser-win32-x64.zip",
+    "hash": "sha1:d68f1944864d14f1ff49282e1169898d9f993ec0",
```

Comment:
Why `sha1` when there is no hash extraction?

Reply:
Got schema error without sha1:

Reply:
Use sha256 hash

Reply:
As soon as you resolve all comments just run `.\bin\checkver.ps1 oss-browser -f` and hashes will be automatically calculated.

Reply:
Got it, thx